src/scripts/main.js

```diff
@@ -2,13 +2,34 @@
 
 function waitFor(element, eventName) {
 // write your code here
+  return new Promise(resolve => {
+    // eslint-disable-next-line no-shadow
```

Comment:
Do you really need this linter disabling?
I don't see anything that should require this action.

Reply:
If you don't use `event` argument, you can remove it at all. Just use callback function without any arguments.